### PR TITLE
add: fox and pat command

### PR DESCRIPTION
**Pull request by AdityaTD**

### CODE REVIEW

File: commands/Misc/fox.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { Command } = require('klasa');
+const snekfetch = require('snekfetch');
```
Comment (kyranet) [inline commands/Misc/fox.js]: Please use `node-fetch` instead.

Comment (AdityaTD) [reply commands/Misc/fox.js]: Okay let me try

**Comment by KingDGrizzle** [comment]
Bump, as this has requested changes since October 1st..

**Comment by kyranet** [comment]
Closing due to the age of the requested changes, feel free to make a new PR later.

**Pull request closed without merging**